src/app/sketcher_app.cpp

```diff
@@ -0,0 +1,11 @@
+#include <QApplication>
```

Comment:
I posted https://github.com/schrodinger/sketcher/pull/3 with all the skeleton code I put together; let's check that in separately to make this review easier on the(my) eyes

Reply:
ok